.github/workflows/auto-add-to-project.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     name: Add issue and bugs to project
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/add-to-project@v0.5.0
```

Comment:
Let me update this across all plugins right away. We'll upgrade straight to 1.0.2: https://github.com/actions/add-to-project/releases/tag/v1.0.2
Same for actions/checkout@v4 below.